hier_config/constructors.py

```diff
@@ -215,9 +215,7 @@ def _config_from_string_lines_end_of_banner_test(
 
 
 def _load_from_string_lines(config: HConfig, config_text: str) -> None:  # noqa: C901
-    if isinstance(config.driver, HConfigDriverJuniperJUNOS):
-        config_text = _convert_to_set_commands(config_text)
-
+    config_text = config.driver.config_preprocessor(config_text)
```

Comment:
I have some work in flight that's related to this idea. Instead of a function that returns a new string, it takes the raw config and returns lines. My intent is to simplify this function and move banner processing to the driver. I think this is going in the right direction though and I'm good with it. Its possible it might change again as we get this functionality nailed down which is okay.